README.md

```diff
@@ -1,7 +1,18 @@
 # About
 
+[![Build Status](https://travis-ci.org/openaire/vipe.png?branch=master)](https://travis-ci.org/openaire/vipe)
+
+
 This is a tool for visualizing Apache Oozie pipelines.
 
 # License
 
 The code is licensed under Apache License, Version 2.0
+
+# Dependencies
+
+Python packages that the application depends on are listed in the `requirements.txt` file. Note that `pyyaml` package requires `libyaml` library to be installed in the system. On Ubuntu system, this can be installed by running `apt-get install libyaml-dev`
+
```

Comment:
It is worth mentioning about using `pip3` instead of `pip` and installing python `test` library by executing: `sudo pip3 install pytest` no avoid confusions and troubles when installing `python-pytest` via `apt-get`.